lib/modules/modules.go

```diff
@@ -205,7 +205,7 @@ func ValidateResource(res types.Resource) error {
 	case types.AuthPreference:
 		switch r.GetSecondFactor() {
 		case constants.SecondFactorOff, constants.SecondFactorOptional:
-			return trace.BadParameter("cannot disable two-factor authentication on Cloud")
+			// return trace.BadParameter("cannot disable two-factor authentication on Cloud")
```

Comment:
I'm doing the same, but we cannot commit this change 😅

Reply:
god...